README.md

```diff
@@ -1,60 +1,95 @@
-  ```
-  
-  Guide:
-  
-  ```
-Usage: ./bin/fft.out beta [path]
-
-beta - roughness factor
-
-path - optional - path to a square input image, dimensions are a power of 2.
-  ```
-  #### Clean Up
-  In order to clean the bin directory, run ```make clean```.
-
-  In order to remove only the generated images, run ``` make cleanimg```.
-</details> 
-
-## Results
-Beta                       |Original                   |  Filtered Discrete Fourier Transform     | 1/f Noise generated by Inverse Fourier Transform
-:-------------------------:|:-------------------------:|:----------------------------------------:|:------------------------------------------------:
-1.6                        |![](samples/download.png)  |  ![](docs/dft_16.png)                   | ![](docs/ifft_16.png)
-2.4                        |![](samples/download.png)  |  ![](docs/dft_24.png)                   | ![](docs/ifft_24.png)
-
-
-## References
-- [2D FFT](https://github.com/karimnaaji/fft)
-- [Fractal Noise](http://paulbourke.net/fractals/noise/)
-- [Fractal Landscapes via FFT](https://www3.risc.jku.at/education/courses/ws2016/cas/landscape.html)
-- [Dickerson Terrain](https://web.williams.edu/Mathematics/sjmiller/public_html/hudson/Dickerson_Terrain.pdf)
-- [Image filtering in the frequency domain](http://people.ciirc.cvut.cz/~hlavac/TeachPresEn/11ImageProc/13FourierFiltrationEn.pdf)
-- [Image Enhancement in the
-Frequency Domain](http://www.cs.tut.fi/~moncef/SGN-3016-DIP/Chap04.pdf)
-- [Highpass filtering](https://eecs.wsu.edu/~cs445/Lecture_9.pdf)
-- [Moving into the Frequency Domain](http://users.cs.cf.ac.uk/Dave.Marshall/CM0268/PDF/09_CM0268_Frequence_Space.pdf)
-
-#### Special thanks to [Faraz Shaikh](https://github.com/FarazzShaikh) for introducing me to noise generation and terrain building 🤯
+  <p align="center">
+    2D Fractal Noise using Fast Fourier Transforms.
+    <br />
+    <a href="https://github.com/IamShubhamGupto/FFT-Noise/blob/main/docs/ifft_24.png">View Demo</a>
+    ·
+    <a href="https://github.com/IamShubhamGupto/FFT-Noise/issues/new">Report Bug</a>
+  </p>
+</p>
+
+# What is this?
+
+FFT-Noise is an implementation of a method of producing Fractal Noise using Fast Fourier Transforms based on a method found in an [HRUMC Presentation](https://web.williams.edu/Mathematics/sjmiller/public_html/hudson/Dickerson_Terrain.pdf) by **Krista Bird**, **Thomas Dickerson**, and **Jessica George**<sup>1</sup>.
+
+It takes in an array of random points as input (possibly from an image) and returns FFT-Noise generated from it. The Noise looks something like this:
+
+<p align="center">
+    <img src="./Assets/ifft_24.png" alt="FFT-Noise Sample">
+</p>
+
+This noise can be used in procedural texture and terrain generation.
+
+# How does this work?
+
+According to _Techniques for Fractal Terrain Generation_<sup>1</sup> FFT-Noise is generated in four distint steps:
+
+- Obtain an array of (random valued) point.
+- Perform an FFT on the points.
```

Comment:
Are you sure you call this step FFT ? From my understanding FFT is the process of converting from real to real and imaginary and then converting that back. I think this should be Discrete Fourier Transform (DFT)

Reply:
FFT is a faster way of perform a DFT. Besides in this implementation we are using a FFT and IFFT function.